configuration-service/handlers/service.go

```diff
@@ -143,7 +143,46 @@ func PutProjectProjectNameStageStageNameServiceServiceNameHandlerFunc(params ser
 
 // DeleteProjectProjectNameStageStageNameServiceServiceNameHandlerFunc deletes a service
 func DeleteProjectProjectNameStageStageNameServiceServiceNameHandlerFunc(params service.DeleteProjectProjectNameStageStageNameServiceServiceNameParams) middleware.Responder {
-	return middleware.NotImplemented("operation service.DeleteProjectProjectNameStageStageNameServiceServiceName has not yet been implemented")
+	logger := utils.NewLogger("", "", "configuration-service")
+
+	common.LockProject(params.ProjectName)
+	defer common.UnlockProject(params.ProjectName)
+
+	projectConfigPath := config.ConfigDir + "/" + params.ProjectName
+	servicePath := projectConfigPath + "/" + params.ServiceName
+
+	if !common.StageExists(params.ProjectName, params.StageName, false) {
+		return service.NewDeleteProjectProjectNameStageStageNameServiceServiceNameDefault(500).WithPayload(&models.Error{Code: 500, Message: swag.String("Stage  " + params.StageName + " does not exist.")})
```

Comment:
Should we return here a 400?

Reply:
agreed, i have adapted the error code for that case